### PR TITLE
feat: track number of CLI installations [FTL-8744]

### PR DESCRIPTION
**Pull request by YoussefAWasfy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "prepare:dev": "is-ci || (husky install && npm run prepare-commit-hooks)",
     "prepare": "node -e \"process.exit(process.env.NODE_ENV === 'production' ? 0 : 1)\" || npm run prepare:dev",
     "remove:dist": "shx rm -rf dist",
-    "generate:api": "node scripts/generate-api.cjs"
+    "generate:api": "node scripts/generate-api.cjs",
+    "postinstall": "sh \"bin/postinstall\""
```
Comment (dendavidov-affinidi) [inline package.json:35]: not cross-platform

File: bin/postinstall [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+npm_config_global=$(npm config get global)
+if [ $npm_config_global = "true" ]; then
+code=$(curl --write-out "%{http_code}\n" --location -X POST 'https://analytics-stream.prod.affinity-project.org/api/events' \
```
Comment (dendavidov-affinidi) [inline bin/postinstall:5]: what if there is no curl?

**Review by dendavidov-affinidi — CHANGES_REQUESTED** [review]
This solution is not cross-platform.
I'd recommend to use js file instead of bash
example:
https://github.com/zloirock/core-js/blob/master/packages/core-js/package.json#L68

**Review by JonasEtzold — APPROVED** [review]
LGTM

**Pull request merged**